### ISSUE TITLE
Accessibility Support

### ISSUE THREAD

**Issue by jonecx** [thellmund/Android-Week-View#105]
Calendar events are not recognized by accessibility focus.

Clicking on calendar events doesn't respond while Accessibility is on

**Comment by thellmund** [comment]
I’ll make sure to add this in an upcoming release.

**Comment by thellmund** [comment]
I started working on accessibility support in the `th/accessibility` branch. You can tap on events and on empty days. In the latter case, TalkBack will simply read the corresponding date. 

I still need to change it so that it reads the exact time that the user tapped when clicking on an empty date. Besides that, I think it’s pretty much complete. 

You can try it out if you switch to `implementation 'com.github.thellmund:Android-Week-View:th~accessibility-SNAPSHOT'` in your `build.gradle`. If you think there’s anything missing, let me know.

**Comment by thellmund** [comment]
Added in #127.
